priv/package.json

```diff
@@ -0,0 +1,4 @@
+{
+  "name": "elixir-nodejs",
+  "type": "commonjs"
+}
```

Comment:
I think it's looking for a new line at the end here.

Reply:
Yes, right. That's how Github shows lack of newline at the end of the file. I'll add it

Reply:
Done